Documentation/TypoScriptConfiguration/Index.rst

```diff
@@ -122,7 +122,8 @@ First, extend :ts:`// Part 1: Fluid template section` by the following lines::
       templateName = TEXT
       templateName.stdWrap.cObject = CASE
       templateName.stdWrap.cObject {
-         key.data = pagelayout
```

Comment:
The field does not exist, but the data exists as it's calculated considering the inheritance: https://github.com/TYPO3/TYPO3.CMS/blob/9.5/typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php#L4977 It's also documented at: https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/DataTypes/Index.html#pagelayout

What's the reason to think it's not available? Maybe there is some bug or misunderstanding. Or some other issue with the example?